### PR TITLE
New winston logger configuration

### LINKED ISSUE #262: Use winston logger.configure to improve logger configuration

### PR DESCRIPTION
**Pull request by mamaso**
resolves #262

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -33,13 +33,13 @@ var loadConfiguration = require('./configuration'),
         pageSize: 50,
         logging: {
             level: environment.debug ? 'debug' : 'info',
-            transports: {
-                Console: {
+            transports: [
+                new (require('winston').transports.Console)({
```
Comment (danderson00) [inline src/index.js]: Move require statement to top of file. Important to make dependencies for each module obvious.

**Comment by danderson00** [comment]
Where is the logger configured from now?

**Comment by mamaso** [comment]
express/index.js

**Comment by danderson00** [comment]
Where abouts? I don't see it - just the removed call to configure.

**Comment by mamaso** [comment]
https://github.com/Azure/azure-mobile-apps-node/blob/master/src/express/index.js#L42

**Comment by danderson00** [comment]
That line is removed from your branch - https://github.com/mamaso/azure-mobile-apps-node/blob/logger/src/express/index.js#L42

**Comment by mamaso** [comment]
lol man I'm failing hard today, good catch.. yeesh

**Comment by danderson00** [comment]
hehe Monday

**Pull request merged**